USAGE.rst

```diff
@@ -0,0 +1,108 @@
+=================================
+NCI SEER Pediatic WSI Pilot Usage
+=================================
+
+See README.rst for installation.
+
+Importing Data
+==============
+
+TODO: clean this up when the process is worked out
+
+- Add image files and the metadata excel file to the import directory on the local filesystem
+- Run the import process
+- TODO: describe structure for file/folder layout and csv file, which should go into import folder for data ingest
+- TODO: describe the file renaming
+- The imported files will now appear in the DSA in the ``SEER`` collection, in the ``Imported`` folder
+
+
+Exporting Data
+==============
+
+TODO: clean this up when the process is worked out
+
+- When images are in the ``SEER`` collection, in the ``Finished`` folder, they can be exported
+- Run the export process
+- The exported files will now appear on the local filesystem in the export directory
+
+
+Redaction
+=========
+
+Many of the workflow states provide controls to allow the user to indicate PHI that should be redacted, staging that PHI for processing.
+
+The user can inspect the image and metadata for PHI, can mark individual metadata fields for redaction from the ``imported`` or ``quarantine`` state, and can indicate if any of the non-primary images should be redacted. When all PHI has been staged for redaction, the user can click the ``Process`` button, which will make a copy of the existing image and place that copy in the ``original`` state, ond will move the image to the ``processed`` state. As part of moving the data to the ``processed`` state, the metadata fields and associated images marked for redaction will be deleted.
+
+TODO: deleted or overwritten or changed??
```

Comment:
All of the files we are currently handling are variants of TIFF.  When a field is redacted change it (e.g., titles and dates), the original value is completely replaced with the new value.  When a field or image is redacted completely (any other field), it is removed.  Label images that are redacted are replaced with an image that has the item's new name (this will be the ImageID).

Reply:
> Label images that are redacted are replaced with an image that has the item's new name (this will be the ImageID).

what is the image that replaces the label image in this case?

Reply:
It is just a black image that contains the item's name.  E.g.,
![image](https://user-images.githubusercontent.com/8781639/91068339-2303dc00-e602-11ea-8e3b-dc7e88557995.png)